.travis.yml

```diff
@@ -9,6 +9,7 @@ env:
 branches:
   only:
     - master
+    - next
```

Comment:
This whole `branches` block should be removed to allow all PRs to be build (including this one so we can directly see any improvements if any in this PR). I don't know why that was added anyways, doesn't really make sense.